mindsdb_native/libs/controllers/transaction.py

```diff
@@ -223,10 +223,13 @@ def _run(self):
             self.save_metadata()
             self._call_phase_module(module_name='ModelInterface', mode='train')
 
+            MODEL_ANALYZER_COLUMNS_LIMIT = 100
```

Comment:
For now this will do, but I feel like it might be too high a limit. We should find more datasets on the bigger side and see what's the best threshold.